packages/ui/src/kit/VAccordion/VAccordionItem.theme-default.scss

```diff
@@ -0,0 +1,46 @@
+.v-accordion-item.theme-default {
+  cursor: pointer;
+  border-top: 1px solid #dadde4;
+  transition: all 150ms ease-in-out;
+
+  &:last-child {
+    border-bottom: 1px solid #dadde4;
+  }
+
+  &:hover {
+    background-color: rgba(213, 217, 224, 0.25);
```

Comment:
this color is --endpass-ui-color-grey-2 with 0.25 opacity, please use our defined colors

Reply:
It's difficult to determine what's the difference between grey-2 and grey-3, especially when in Figma impossible to get color and no labels which may point to correct from predefined ones.

Reply:
@ifiri we use --endpass-ui-color-grey-2 and we have reasons for that. Please, use it too

Reply:
@kazurus I understand it's reasons, I use variables as well. It's good practice. I mean naming convention and impossibility to determine color in Figma.